tokio-fs/src/file/mod.rs

```diff
@@ -204,6 +205,9 @@ impl File {
     ///     tokio::run(task);
     /// }
     /// ```
+    #[deprecated(since = "0.1.5", note = "use tokio::io::AsyncSeek instead")]
+    #[doc(hidden)]
+    #[allow(deprecated)]
     pub fn poll_seek(&mut self, pos: io::SeekFrom) -> Poll<u64, io::Error> {
```

Comment:
Is this not going to cause fn name conflicts?

Reply:
The deprecated fn is saying to use the fn on the `AsyncSeek` trait, however, I'm pretty sure inherent methods take precedence of trait fns, so it would not be possible to call the fn on the trait. Or maybe rustc will fail with an "ambiguous fn" error.

Reply:
Ah, I think you're right. I'm not sure how to deal with this, though: `poll_seek` is pretty clearly the name the function should have to line up with the other traits. Removing this function would be a breaking change, and naming it something else just punts another breaking change down the road.

Reply:
Yeah, I'm not sure either. Are you able to add a test that uses `File::poll_seek` while `AsyncSeek` is in scope to see what happens?